### PR TITLE
PIM-5460  Several datepickers: remove jQuery UI datepickers, keep Bootstrap's

### PR DESCRIPTION
**Pull request by aRn0D**
| Q                    | A
| ----------------- | ---
| Specs             | No
| Behats            | No
| Blue CI            | TODO
| Changelog updated | Yes
| Review and 2 GTM  | Yes

Some Skeleton's commits have been "cherry picked" from the master. The bootstrap datepicker configuration (data attribute) has been added to the date form type html markup.

### CODE REVIEW

File: src/Pim/Bundle/UIBundle/Resources/views/Form/pim-fields.html.twig [hunk 1]
```diff
@@ -394,7 +394,7 @@
 {% endblock %}
 
 {% block oro_date_widget %}
-    {% set attr = attr|merge({ 'class': (attr.class is defined ? attr.class ~ ' ' : '') ~ 'datepicker', 'placeholder': placeholder|trans }) %}
+    {% set attr = attr|merge({ 'class': (attr.class is defined ? attr.class ~ ' ' : '') ~ 'datepicker', 'data-provide': 'datepicker', 'data-date-format': 'yyyy-mm-dd', 'placeholder': placeholder|trans }) %}
```
Comment (juliensnz) [inline src/Pim/Bundle/UIBundle/Resources/views/Form/pim-fields.html.twig]: good catch dude :)

**Comment by juliensnz** [comment]
Looks nice to me. GTM

File: src/Pim/Bundle/UIBundle/Resources/config/requirejs.yml [hunk 2]
```diff
@@ -48,25 +42,20 @@ config:
...
-            jquery-ui-timepicker: jquery-ui-timepicker
 
     paths:
         text:                          pimui/lib/text/text
         json:                          pimui/lib/json2/json2
         base64:                        pimui/lib/base64/base64
         jquery:                        pimui/lib/jquery/jquery-1.10.2
-        jquery-ui:                     pimui/lib/jquery-ui/jquery-ui.min
-        jquery-ui-timepicker:          pimui/lib/jquery-ui-timepicker-addon
+        jquery-ui:                     pimui/lib/jquery-ui/jquery-ui-1.11.4.custom.min
```
Comment (jmleroux) [inline src/Pim/Bundle/UIBundle/Resources/config/requirejs.yml]: what is this *custom* version ?

**Comment by aRn0D** [comment]
@skeleton We use the right datepicker now, could you review it.

File: src/Pim/Bundle/UIBundle/Resources/config/requirejs.yml (hunk 2 above)
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/config/requirejs.yml]: custom version contains all jquery-ui components except Accordion, Datepicker, Progressbar and Tabs (which are manage by bootstrap components)

**Comment by skeleton** [comment]
GTM :)

**Comment by aRn0D** [comment]
@juliensnz Is it still "GTM" ?

**Comment by juliensnz** [comment]
Yup

**Comment by aRn0D** [comment]
See http://core-ci.akeneo.com/job/dci_behat/1284 for ce build.

**Pull request merged**